git-extra/vimrc

```diff
@@ -14,7 +14,16 @@ set clipboard=unnamed           " set clipboard to unnamed to access the system
 set wildmode=list:longest,longest:full   " Better command line completion
 
 " Show EOL type and last modified timestamp, right after the filename
-set statusline=%<%F%h%m%r\ [%{&ff}]\ (%{strftime(\"%H:%M\ %d/%m/%Y\",getftime(expand(\"%:p\")))})%=%l,%c%V\ %P
+" Set the statusline
+set statusline=%f               " filename relative to current $PWD
+set statusline+=%h              " help file flag
+set statusline+=%m              " modified flag
+set statusline+=%r              " readonly flag
+set statusline+=\ [%{&ff}]      " Fileformat [unix]/[dos] etc...
+set statusline+=\ (%{strftime(\"%H:%M\ %d/%m/%Y\",getftime(expand(\"%:p\")))})  " last modified timestamp
+set statusline+=%=              " Rest: right align
+set statusline+=%l,%c%V         " Position in buffer: linenumber, column, virtual column
+set statusline+=\ %P            " Position in buffer: Percentage
```

Comment:
Nicely done!

Since this is *expanding* rather than *simplifying*, let's call it that in the commit message?

Also, I see that `%<` was removed. Can you describe in the commit message, please, what this does and why it is okay to remove it?

Reply:
Sure, will do.

Reply:
Thanks!